pkg/api/routes.go

```diff
@@ -814,6 +814,10 @@ func (rh *RouteHandler) DeleteManifest(response http.ResponseWriter, request *ht
 		} else if errors.Is(err, zerr.ErrBadManifest) {
 			zcommon.WriteJSON(response, http.StatusBadRequest,
 				apiErr.NewErrorList(apiErr.NewError(apiErr.UNSUPPORTED, map[string]string{"reference": reference})))
+		} else if errors.Is(err, zerr.ErrBlobReferenced) {
```

Comment:
Can you pls check why Blob is referenced here? This is a manifest delete path right? not blob?

Reply:
ErrManifestReferenced

Reply:
Also add a comment that this manifest could be inside an index hence the conflict